src/main/java/io/ipfs/api/IPFS.java

```diff
@@ -490,6 +492,28 @@ public Object log() throws IOException {
         }
     }
 
+    private boolean versionIsValid(String version) {
+        String[] parts = version.split("\\.");
```

Comment:
This library needs some love and updating in general I will try and devote some time asap.